specs/models/Step.schema.json

```diff
@@ -132,6 +132,56 @@
     },
     "name": {
       "type": "string"
+    },
+    "properties": {
```

Comment:
I think it'd be better to make a MappingStep.schema.json that uses the example at the end of this comment - https://github.com/json-schema-org/json-schema-spec/issues/348#issuecomment-322940347 - to grab the Step properties via "allOf". We can do likewise for the other concrete types of steps when we define schemas for those.

Reply:
I think that's okay for now. We can see if a springfox upgrade fixes it later.